lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
because type string is typing too soon

https://openqa.suse.de/tests/569123#step/smt/4